### PR TITLE
feat: adding responsive h1

### PR DESCRIPTION
**Pull request by alisedarta**

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -89,13 +98,18 @@ line-height: 24px;
...
+    font-size: 48px;
+    line-height: 56px;
+    }
+}
 
-      h1{
-        font-size: x-large;
-        color: red;
-      }
+@media (min-width: 835px){
```
Comment (osartun) [inline src/index.css:109]: Btw: I could've been more careful when choosing the frames for the different viewport sizes in Figma 😅 
835px is a very uncommon breakpoint.
You can see here which breakpoints would've been more common: https://www.browserstack.com/guide/responsive-design-breakpoints

File: src/index.css [hunk 2]
```diff
@@ -89,13 +98,18 @@ line-height: 24px;
...
+  h1{
+    margin-left: 32px;
+    font-size: 48px;
+    line-height: 56px;
+  }
```
Comment (osartun) [inline src/index.css:114]: Although, actually I notice something: This is the desktop CSS. However, you're already applying it for viewports 1px wider than the tablet frame in Figma.

The frames in Figma usually represent the smallest viewport widths. So, in this project:
* Mobile CSS would be applied up to 833px
* Tablet CSS would be applied from 834px to up to 1511px
* Desktop CSS would be applied from 1512px

Currently, you're applying it like this:
* Mobile CSS is applied up to 430px
* Tablet CSS is applied from 431px to up to 834px
* Desktop CSS is applied from 835px

**Review by osartun — COMMENTED** [review]
I left a few comments. However, something I missed is: There are other text styles which also change.

* Title ← This is the one you took care of
* Artwork title ← This changes as well, but it's not implemented yet
* Artwork artist ← Same

File: src/index.css (hunk 1 above)
Comment (alisedarta) [reply src/index.css:109]: OK, I'll keep it in mind & maybe add to my learnings, but for now I'll keep it as figma designs (unless you object haha)

File: src/index.css [hunk 3]
```diff
@@ -6,6 +6,15 @@ body {
   background-color: #F5F3F1;
 }
 
+h1 {
+  margin-top: 72px;
+  margin-left: 16px;
```
[the remark this answers is not in the document]
Comment (alisedarta) [reply src/index.css:13]: i've set body left & top margings now to 0

File: src/index.css [hunk 4]
```diff
@@ -57,7 +68,7 @@ body {
 .card-title {
 display: inline;
 text-align: left;
-font-size: 24px;
+font-size: 20px;
```
Comment (osartun) [inline src/index.css:71]: The `.artist-name` also has a `font-size` of 20px on mobile

**Review by osartun — APPROVED** [review]
One more adjustment, but it's looking good!
Approved 👍

File: src/index.css (hunk 4 above)
Comment (alisedarta) [reply src/index.css:71]: thank you, adjusted. I shall be more careful

Comment (osartun) [reply src/index.css:71]: No worries
This comes with time
Then you know more and more what to look out for

**Pull request merged**